site/content/articles/openshift-monitoring-guide.adoc

```diff
@@ -48,35 +48,37 @@ Docker is an essential component of an OpenShift environment. The overall health
 
 |Docker Daemon |Check that docker is running on a system | `systemctl is-active docker`
 
-|Docker Storage|Check that docker's storage has adequate space | `echo $(echo \"$(docker info 2>/dev/null \| awk '/Data Space Available/ {print $4}') / $(docker info 2>/dev/null \| awk '/Data Space Total/ {print $4}')\" \| bc -l) '>' 0.3 \| bc -l`
+|Docker Storage|Check that docker's storage has adequate space | `echo $(echo \"$(docker info 2>/dev/null  &#124; awk '/Data Space Available/ {print $4}') / $(docker info 2>/dev/null  &#124; awk '/Data Space Total/ {print $4}')\"  &#124; bc -l) '>' 0.3  &#124; bc -l`
```

Comment:
What values are these supposed to return. In my test queries, both returned `1` with an exit code of 0

Reply:
Had to split this into 2 different commands.  1 command is for devicemapper the other is for overlay2.  The original command would not work on a system configured with overlay2.